### DISCUSSION TITLE
XMAX3 SOFTWARE OUTDATED AND UNUSABLE FOR KLIPPER

### DISCUSSION THREAD

**Discussion by Badghostuk** [QIDITECH/QIDI_MAX3#64] [Ideas]
The version of Moonraker is too outdated to even complete simple tasks like connect to server etc why can ALL other manufacturers get KLIPPER correct but QIDI Cant

**Comment by arielcoralv2023-oss** [comment]
new machin and update dont work i have 2 qidi 3max3 and not working and the update v43.15 do noting or fix  the problem

**Reply by Broncorniclops** [reply]
Look into Freedi, it will get your printer up to date.
